indico/MaKaC/webinterface/pages/static.py

```diff
@@ -47,7 +47,7 @@ class WPStaticTimetable(WPStaticEventBase, WPDisplayTimetable):
 
 
 class WPStaticConferenceProgram(WPStaticEventBase, WPConferenceProgram):
```

Comment:
I don't think this will work. This uses a legacy WP which gets the data from legacy.

https://github.com/indico/indico/blob/master/indico/MaKaC/common/offlineWebsiteCreator.py#L314

You can remove the WP handling there (program is the only remaining one) and add the new RH instead. Then create a new WP similar to the other "new" static WPs here.